src/ProjectTemplates/test/Infrastructure/Directory.Build.props.in

```diff
@@ -1,6 +1,6 @@
 <Project>
 <!-- This file gets copied above the template test projects so that we disconnect the templates from the rest of the repository -->
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

Comment:
This and src/ProjectTemplates/test/Infrastructure/TemplateTests.props.in are also template files. Should change ProjectTemplates.Tests.csproj to pass the default TFM in.

Should also use the substituted value at line 40.